x-pack/solutions/search/plugins/enterprise_search/server/deprecations/pre_eight_index_deprecator.ts

```diff
@@ -8,11 +8,17 @@
 import { ElasticsearchClient } from '@kbn/core/server';
 
 const ENT_SEARCH_INDEX_PREFIX = '.ent-search-';
+const ENT_SEARCH_DATASTREAM_PATTERN = [
+  'logs-enterprise_search.*',
+  'logs-app_search.*',
+  'logs-workplace_search.*',
+  'logs-crawler-*',
```

Comment:
Sorry for the late thoughts... I am a bit concerned about this pattern... Is it conceivable that users may have data streams NOT created/owned by enterprise search that may match this pattern?

Reply:
These are for the App Search crawler, that is from 7.x onwards... the managed crawler has the `logs-elastic_crawler-*` prefix.. .;)